packages/bella/config/index.js

```diff
@@ -52,7 +52,7 @@ export default {
     shoulderSlopeBack: 1.23,
     neckWidthBack: 0.197,
     neckWidthFront: 0.17,
-    shoulderToShoulderCorrection: 0.995,
+    // shoulderToShoulderCorrection: 0.995,
```

Comment:
Ok to remove?
```suggestion
```

Reply:
I left it in place, commented out, for history sake. I like to keep removals of code in the file for a while as an easy indication of what was changed. I guess with github keeping track, we don't really have to.

Reply:
My preference is to never allow commented code but I am happy to follow an existing convention.

Reply:
I have no idea if there is a convention. ;-)

Reply:
Code style is enforced by prettier, but comments are comments. They will not be removed.

Git is where history lives. I don't mind leaving comments in while things are a WIP, but I also feel we should strip them prior to merging.